pkgs/development/libraries/phash/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, pkgconfig, cimg, imagemagick }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "pHash";
+  version = "0.9.4";
+
+  buildInputs = [ cimg ];
+
+  # CImg.h calls to external binary `convert` from the `imagemagick` package
+  # at runtime
+  propagatedBuildInputs = [ imagemagick ];
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  configureFlags = ["--enable-video-hash=no" "--enable-audio-hash=no"];
+  postInstall = ''
+    cp ${cimg}/include/CImg.h $out/include/
+  '';
+
+  src = fetchFromGitHub {
+    owner = "clearscene";
+    repo = "pHash";
+    rev = "0.9.4";
+    sha256 = "0y4gknfkns5sssfaj0snyx29752my20xmxajg6xggijx0myabbv0";
+  };
+
+  meta = {
```

Comment:
style nit: common pattern is to do:
```
meta = with stdenv.lib; { }
```
so you don't have to fully qualify everything in the meta